Adding Prague 2019 landing page.

mxsasha opened the pull request:
This is my attempt at setting up the Prague 2019 landing page. It works, but there may be conventions that I overlooked. I started out from the Prague 2018 website. I don't know whether I'm supposed to remove all other content other than the landing page from conf/prague/2019.

I don't know if we usually do a "dates announced" announcement, but apparently the news posting list generator needs at least one item in the ``prague-2019`` tag, or it'll publish all general news.

Also, there appears to be a reference somewhere to `_templates/include/2019/prague-sponsors.html` and the build would break without it. I haven't been able to track down where this reference came from, so I made that an empty file right now to get it to build

mxsasha committed:
Adding Prague 2019 landing page.

plaindocs commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,108 @@
+templatecode: eu # HACK
+year: 2019
+city: Prague
+local_area: Czech Republic
```
```suggestion
local_area: Czechia
```

plaindocs commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,108 @@
+  writing_day:
+    price: TBD
+
+date: # how do we handle these? human readable would be nice.
```
I *think* some of these are not quite needed, prob `day_four` as we hack that into a multi day day three :-p

mxsasha replied:
Removed it, seems fine.

plaindocs commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,108 @@
+    Main Ballroom
+  unconfroom:
+    Unconference Room, across from the Main Ballroom
+  photos: https://www.flickr.com/photos/writethedocs/sets/72157695294209550
```
```suggestion

```

plaindocs commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,108 @@
+  unconfroom:
+    Unconference Room, across from the Main Ballroom
+  photos: https://www.flickr.com/photos/writethedocs/sets/72157695294209550
+  videos: https://www.writethedocs.org/videos/prague/2019/
```
```suggestion

```

I think we've might have some pretty basic if "empty don't publish these" logic around somewhere

plaindocs commented on docs/conf/prague/2019/convince-your-manager.rst:
```diff
@@ -0,0 +1,65 @@
+In addition, it can be useful to share some info about previous conferences. You can find the websites for previous events on :doc:`/conf/index/`, and a quick list of last year's talks down below.
+But perhaps more useful might be some of the info in our :doc:`press-kit/`, which includes community testimonials, photos, and more.
+
+List of talks from 2017
```
Wonder if we can `{{year-1}}` all of these 2017s to make them 2018

mxsasha replied:
Yes, we can. With some filename changes that happened in between.

plaindocs commented on docs/include/conf/current.rst:
```diff
@@ -2,7 +2,7 @@
 
 - :doc:`Write the Docs Portland </conf/portland/2019/index>`, May 19-21, **Portland, Oregon**
-- Write the Docs Prague, September 
+- :doc:`Write the Docs Prague </conf/prague/2019/index>`, September 15-17, **Prague, Czech Republic**
```
```suggestion
- :doc:`Write the Docs Prague </conf/prague/2019/index>`, September 15-17, **Prague, Czechia**
```

plaindocs reviewed (approved):
This looks great!

We should *probably* have cloned the Portland 2019 not Prague 2018, to get the latest "improvements" but there probably isn't *that* much in it.

Setting `flaglanding: True` hides the menus, we don't delete the actual pages.

> I don't know if we usually do a "dates announced" announcement, but apparently the news posting list generator needs at least one item in the prague-2019 tag, or it'll publish all general news.

Correct, hence the empty ish venue+dates announce first news item. 

> Also, there appears to be a reference somewhere to _templates/include/2019/prague-sponsors.html and the build would break without it. I haven't been able to track down where this reference came from, so I made that an empty file right now to get it to build

Yeah, those are in `_templates/include/2019`. We don't have a very clean distinction between what are HTML output templates, and what are partial HTML templates. Those ones are particularly weird due to some formatting issues we had when including them in index.rst and sponsors.html

plaindocs committed:
Update docs/_data/config-prague-2019.yaml

Co-Authored-By: mxsasha <github@mxsasha.eu>

plaindocs committed:
Update docs/_data/config-prague-2019.yaml

Co-Authored-By: mxsasha <github@mxsasha.eu>

plaindocs committed:
Update docs/_data/config-prague-2019.yaml

Co-Authored-By: mxsasha <github@mxsasha.eu>

plaindocs committed:
Update docs/include/conf/current.rst

Co-Authored-By: mxsasha <github@mxsasha.eu>

mxsasha committed:
address comments

mxsasha commented:
Addressed all comments (needs squashing when merging).

Perhaps someone could still have a look at https://github.com/writethedocs/www/pull/750/files#diff-adcd8cfa613f456930347b06ec1acc83 - as that's the only new copy I've added.

Regarding files to include: if we include pages with important details like ticket pricing, ticket links or a CFP announcements, those pages are available to anyone that can guess the super predictable URL. Before you know it, a URL like that appears on twitter or slack and looks authentic because it is really on our website. So I trimmed some of it - we can just fill it back in later.

plaindocs commented:
@mxsasha that diff does't resolve for me, wanna double check? 

I hear you on the pricing details etc. We've not really worried about it in the past is all.

mxsasha commented:
> @mxsasha that diff does't resolve for me, wanna double check?

That's weird, the link works for me. It's supposed to link to the change for `docs/conf/prague/2019/news/prague-19-dates.rst`

plaindocs commented:
Yup - looks fine.

plaindocs committed:
Remove old new news

plaindocs commented:
@thatdocslady wanna confirm sponsorship levels?

thatdocslady commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,103 @@
+
+tickets:
+  community:
+    price: €500
```
600EUR

thatdocslady commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,103 @@
+  community:
+    price: €500
+  corporate:
+    price: €250
```
300EUR

thatdocslady commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,103 @@
+  corporate:
+    price: €250
+  independent:
+    price: €150
```
175EUR

thatdocslady commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,103 @@
+
+sponsorship:
+  first_draft:
+    price: €500
```
750EUR

thatdocslady commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,103 @@
+  first_draft:
+    price: €500
+  second_draft:
+    price: €1000
```
1500EUR

thatdocslady commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,103 @@
+  second_draft:
+    price: €1000
+  publisher:
+    price: €3000
```
3500EUR

thatdocslady commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,103 @@
+  publisher:
+    price: €3000
+  patron:
+    price: €6000
```
7500EUR

thatdocslady commented on docs/_data/config-prague-2019.yaml:
```diff
@@ -0,0 +1,103 @@
+  patron:
+    price: €6000
+  keystone:
+    price: €12000
```
13000EUR

thatdocslady reviewed (commented):
Ticket prices and sponsorship rates updated, assume no change if no comment below the line item.

mxsasha committed:
Update pricing.

plaindocs commented:
Then I think we can publish the landing page, and double check the rest later.

thatdocslady commented:
Please wait until I sign the contract on Monday, unless you think soft-launch is ok without a blog post for the next few days :)

> On 13 Dec 2018, at 22:58, Plaindocs <notifications@github.com> wrote:
> 
> Then I think we can publish the landing page, and double check the rest later.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.

plaindocs commented:
Roger that

ericholscher commented:
🎉
